CONTRIBUTING.md

```diff
@@ -4,10 +4,43 @@
 
 Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.  Please see https://git.k8s.io/community/CLA.md for more info
 
-### Contributing A Patch
+## Contributing A Patch
 
 1. Submit an issue describing your proposed change to the repo in question.
 1. The [repo owners](OWNERS) will respond to your issue promptly.
 1. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
 1. Fork the desired repo, develop and test your code changes.
 1. Submit a pull request.
+
+## Development
+Please go through [CSI Spec](https://github.com/container-storage-interface/spec/blob/master/spec.md) and [General CSI driver development guideline](https://kubernetes-csi.github.io/docs/developing.html) to get some basic understanding of CSI driver before you start.
+
+### Requirements
+* Golang 1.15.+
+* [Ginkgo](https://github.com/onsi/ginkgo) in your PATH for integration testing and end-to-end testing
+* Docker 17.05+ for releasing
+
+### Dependency
+Dependencies are managed through go module. To build the project, first turn on go mod using `export GO111MODULE=on`, then build the project using: `make`
+
+### Testing
+* To execute all unit tests, run: `make test`
+* To execute sanity test run: `make test-sanity`
+* To execute integration tests, run: `make test-integration`
+* To execute e2e tests, run: `make test-e2e-single-az` and `make test-e2e-multi-az`
+
+### Release Process
+Please see [Release Process](./docs/RELEASE.md).
+
+**Notes**:
+* Sanity tests make sure the driver complies with the CSI specification
+* EC2 instance is required to run integration test, since it is exercising the actual flow of creating EBS volume, attaching it and read/write on the disk. See [Integration Testing](../tests/integration/README.md) for more details.
```

Comment:
That's "./tests". Single dot. Suggest going to your PR and click on Browse code (  `<>` ) to make sure that all your links work.